package.json

```diff
@@ -5,30 +5,21 @@
     "license": "CC-BY-3.0",
     "repository": "https://github.com/fluid-project/infusion-docs",
     "scripts": {
-        "docpad": "node node_modules/docpad/bin/docpad run --env static --out out",
-        "deploy": "node node_modules/docpad/bin/docpad deploy-ghpages --env static",
-        "pretest": "node node_modules/docpad/bin/docpad generate --env static --out out",
+        "11ty": "mkdir -p out && eleventy --serve",
```

Comment:
You don't need to specify the mkdir. You have already specified the out folder in ".eleventy.js" file and putting it again here may cause consistency issues in case output directory is changed in future. chane it to "11ty": "eleventy --serve", . similar for build script.

Reply:
I did this for the search index file. It is being generated in the output directory and previously the output directory was produced before the generation of the search index but now the output directory is produced after the generation and the path fails to resolve if there is no out/ directory already.